cli/commands/configure/configure.go

```diff
@@ -115,11 +115,11 @@ func Command(cli *cli.SensuCli) *cobra.Command {
 	}
 
 	_ = cmd.Flags().BoolP("non-interactive", "n", false, "do not administer interactive questionnaire")
-	_ = cmd.Flags().StringP("url", "", cli.Config.APIUrl(), "the sensu backend url")
+	_ = cmd.Flags().StringP("url", "", config.DefaultAPIURL, "the sensu backend url")
```

Comment:
What if a user runs `sensuctl configure` after they have initially configured? If they leave these flags blank, wouldn't their current configuration be overwritten by the defaults?

Reply:
Great catch @nikkiki!
I'm going to revert this commit: https://github.com/sensu/sensu-go/pull/2770/commits/06e975d86342bf810ddbdba206b55fa238e0e067; it was a mistake on my part.